### PR TITLE
SWARM-1915: Add section that describes the existing hollow jars

### PR DESCRIPTION
**Pull request by heiko-braun**

### CODE REVIEW

File: docs/concepts/packaging-types.adoc [hunk 1]
```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Build hollow jars
+
+Wildfly Swarm ships three pre-build hollow jar types, for different use cases:
```
Comment (Ladicek) [inline docs/concepts/packaging-types.adoc]: "WildFly", capital F. Also "pre-built".

File: docs/concepts/packaging-types.adoc [hunk 2]
```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
...
+
+* full: The main Java EE related capabilities
+* web: A smaller functional scope, focused on web technologies
+* microprofile: The capabilities belonging the microprofile specification
```
Comment (Ladicek) [inline docs/concepts/packaging-types.adoc]: "belonging to the MicroProfile specifications": missing "to", wrong capitalization of "MicroProfile", plural "specifications"

Also I guess the descriptions are somewhat vague intentionally, but I think we can do a little better. Something like:

- `full`: all the Java EE capabilities
- `web`: limited set of Java EE capabilities, focused on web-related technologies
- `microprofile`: same as the `web` hollow JAR plus the Eclipse MicroProfile capabilities

File: docs/concepts/packaging-types.adoc [hunk 3]
```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Build hollow jars
```
Comment (Ladicek) [inline docs/concepts/packaging-types.adoc]: Also "pre-built" here.

File: docs/concepts/packaging-types.adoc [hunk 4]
```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
...
+
+WildFly Swarm ships three pre-built hollow jar types, for different use cases:
+
+* full: The main Java EE related capabilities
+* web: A smaller functional scope, focused on web technologies
+* microprofile: The capabilities belonging the microprofile specification
```
Comment (kenfinnigan) [inline docs/concepts/packaging-types.adoc]: Maybe the phrase "Functionality defined by all Eclipse MicroProfile specifications"?

Comment (Ladicek) [reply docs/concepts/packaging-types.adoc]: I had a comment here that disappeared for some reason. Not sure if someone seen it. I'll try to repeat.

This one line has these issues: "belonging _to_ the _MicroProfile_ [or Eclipse MicroProfile] specitication_s_".

But in general, I think this list is too vague. I think we could at least write:

- `full`: all the Java EE capabilities
- `web`: limited set of the Java EE capabilities, focused on web-related technologies
- `microprofile`: same as `web` plus all the Eclipse MicroProfile capabilities

Comment (heiko-braun) [reply docs/concepts/packaging-types.adoc]: i've gone with @kenfinnigan suggestion

Comment (Ladicek) [reply docs/concepts/packaging-types.adoc]: Ah and I was totally wrong, our MP hollow JAR _isn't_ a superset of `web`. Thanks :-)

File: docs/concepts/packaging-types.adoc [hunk 5]
```diff
@@ -53,5 +52,23 @@ executing the hollow JAR:
 ----
 $ java -jar myapp-hollow-swarm.jar myapp.war
 ----
-endif::[]
+
+=== Pre-Built hollow jars
```
Comment (Ladicek) [inline docs/concepts/packaging-types.adoc]: Sorry to be a pain, but when it comes to heading capitalization, maybe this should read `Pre-Built Hollow JARs`?

**Review by tradej — CHANGES_REQUESTED** [review]
Please hold the PR, I will submit another PR with suggestions to @heiko-braun's repository .

**Comment by tradej** [comment]
PR submitted at https://github.com/heiko-braun/wildfly-swarm-1/pull/1

**Comment by heiko-braun** [comment]
thanks @tradej !

**Review by Ladicek — APPROVED** [review]

**Review by tradej — APPROVED** [review]

**Comment by heiko-braun** [comment]
Looks like the linux PR job is stuck: Started 17 hr ago
I am going to merge this

**Comment by kenfinnigan** [comment]
Did you also kill the CI job that's stuck?

**Comment by heiko-braun** [comment]
yes

**Comment by kenfinnigan** [comment]
Thanks

**Pull request merged**